### ISSUE TITLE
Project Settings defaults and scene deletion

### ISSUE THREAD

**Issue by IgnacioFer** [tahoma2d/tahoma2d#743]
A couple of things about 'Project Settings' that I thing would make life easier:

1> I think 'Separate assets into scene subfolders' should be ticked by default and the folders created automatically without the confirmation pop-up.

2> given that T2D saves files in not-the-most-user-friendly way, it would be great to have in Project Settings a 'Delete Scene and All its Assets' option, where we can choose a scene to be complete removed from a project and free up space without going on a hunting expedition through the project's folders. This would be specially useful if you get into the habit of using 'Save Scene As' for incremental saving and end up with a lot of unnecessary scenes when you are done (a shortcut for 'Save Scene As' would also be a good thing)

**Comment by artisteacher** [comment]
I disagree with having "separate assets" checked by default - but perhaps T2D could remember that setting. Ie. if it was checked last time you made a project, it would be checked automatically the next time you make a project.

Deleting through T2D is probably on the complicated side since OT was highly optimized for studio use, where many people are using the same files. Since a level can be used in multiple scenes or might share palettes with other scenes/levels, deleting all the assets of a scene could cause unexpected issues for other scenes. I'm not opposed to the idea, but there should be a warning notification when using a "Delete Scene and All its Assets" command - at least the first time.

**Comment by IgnacioFer** [comment]
Yes, it be enough if to carry the separate asset preference from project to project. More annoying imo is that it asks when it creates the new scene folder - but then again it is only once so no big deal - I'm not sure what the point of asking for confirmation is though.

I get it that deleting all assets would be pretty dangerous... it should not be too easy to find and it should definitely recognize whether the assets being deleted are shared in other scenes that are not being deleted and make you confirm - probably twice. 

I'm happy to have found out how to delete scene files from the Browser room to get rid of obsolete incremental saves. Deleting assets its definitely more risky - but it would be good for messy people who end up leaving a lot of unfinished scenes behind. (e.g. after editing a rough version of a scene in the wip edit and realizing that that scene is no good and has to be completely different so that it is easier starting afresh in a new file)

**Comment by IgnacioFer** [comment]
Actually what seems to be working better for me is:

1. create a different project for every scene (leaving 'Separate assets into scene subfolders' unticked). 
2. when I'm finished with a scene (having created in the process a mess of different versions, incremental saves, name changes, etc.), I 'Export \>Export Current Scene' into a new 'final' project folder
3. 'Remove All Unused Levels' on the newly exported scene in the 'final' folder
4. Trash the original wip scene project folder (preferably after backing it up somewhere) 

This, in addition to naming levels when they are first created so they are easier to find in the 'drawings' folder, seems an easier and safer way to manage obsolete files that the new feature I was proposing.

Part of this way of working is having to import palettes from one project to another, and I'm still unclear why palettes are saved in the 'drawings folder' rather than the 'palettes' folder, or why when I 'Load Color Model' (shouldn't that be 'Import Palette', nested under 'Import'?), I have to pick a .tlv file instead of a .tpl file. I'm sure there are pipelines where all this makes sense but I think the whole Palette saving/loading could do with some extra work to make it more intuitive.

**Comment by IgnacioFer** [comment]
So I just realized 'Remove All Unused Levels' is not '**Delete** All Unused Levels', which is what I thought I was doing (the lack of confirmation dialog should've  been a big hint, but there you go, I feel foolish now)

My feature request now boils down to this: it'd be extremely useful to have a 'Delete All Unused Levels' in the right-click menu of the currently loaded scn in the Browser Room, followed by a confirmation dialog box warning you of the possible repercussions for other scns in the same project.

This feature would of course make more sense for projects created with 'separate assets into scene sub-folders', as it should only delete the unused levels in the folder in which that scene's levels are located. So perhaps in order to prevent accidental deletions of shared levels this option could be greyed out in projects which do not have 'separate assets into scene sub-folders', and only be operational in a scene sub-folder but not in the common project folders.

However implemented, a way to quickly delete unused material once you reach your final file would be a very good addition.
